docker-compose.prod.yml

```
@@ -0,0 +1,100 @@
+services:
+  n8n:
+    environment:
+      - WEBHOOK_URL=${N8N_WEBHOOK_URL:?N8N_WEBHOOK_URL is required}
+      - N8N_ENCRYPTION_KEY=${N8N_ENCRYPTION_KEY:?N8N_ENCRYPTION_KEY is required}
+      - N8N_HOST=n8n.speedandfunction.com
+  opensearch:
+    environment:
+      - OPENSEARCH_PORT=${OPENSEARCH_PORT:?OPENSEARCH_PORT is required}
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)